stubs/EntityManagerInterface.stub

```diff
@@ -68,4 +68,13 @@ interface EntityManagerInterface extends ObjectManager
 	 */
 	public function getClassMetadata($className);
 
+	/**
+	 * @param-immediately-invoked-callable $func
+	 * @param Closure(): T $func
+	 * @return T
+	 *
+	 * @template T
```

Comment:
You have some build errors here. Every callable should be accepted, but you're just allowing Closure here. Change it to `callable(): T`.

Reply:
Also - this wouldn't be a PHPStan error, I'm just pointing out what I'd prefer here.

The errors are reported because you don't have `use Closure` at the top or `\Closure` in the PHPDoc :)